doc/decisions/search_plants.md

```diff
@@ -90,11 +93,25 @@ Cons:
 
 ## Decision
 
+Given the constrains and assumptions Partial Match with String Similarity (pg_trgm) seems to be the best option for us.
+
 ## Rationale
 
+As there is a relatively small number of rows and search terms performance isn't that much of an issue.
+
+The ability to handle spelling mistakes and loose matches is also great for us as plant names can be somewhat complicated to spell.
+
+Generally it is seems to fit best for our usecase.
+Something like elasticsearch could cover the usecase better, however it adds a lot of complexity.
+
 ## Implications
 
--
+The following points are difficult if not impossible to implement using PostgreSQL.  
+Before trying them it should be evaluated if a different database like elasticsearch might be a better fit for this usecase.
+
+- Other columns can be matched with extra syntax (e.g. environmental fit or ecological value)
+- Search in hierarchy is difficult
+- Search accuracy (stop words, stemming, etc.)
```

Comment:
Is this maybe irrelevant because of the spell correction? Wouldn't e.g. apricots get a fix to find apricot?

Reply:
I simply copied that from the use case Benjamin wrote (and as it isn't covered by the current solution and difficult do do I added it here).
The stemming itself might work with spell correction, however it is still a different concept.
Stop words aren't really a probelm as well as they should occur where we are searching.

I can remove this line if you want.